src/measurement/position_measurement.js

```diff
@@ -427,7 +427,7 @@ export function coordsChar(cm, x, y) {
 function wrappedLineExtent(cm, lineObj, preparedMeasure, y) {
   let measure = ch => intoCoordSystem(cm, lineObj, measureCharPrepared(cm, preparedMeasure, ch), "line")
   let end = lineObj.text.length
-  let begin = findFirst(ch => measure(ch).bottom < y, end - 1, 0) + 1
+  let begin = findFirst(ch => measure(ch).bottom <= y, end - 1, -1) + 1
```

Comment:
Doesn't this mean you're giving `findFirst` permission to measure the position of character -1, which isn't something that actually has meaning?